app/src/main/java/com/discordtime/gpts/remote/FirestoreAPI.kt

```diff
@@ -2,11 +2,32 @@ package com.discordtime.gpts.remote
 
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
+import com.discordtime.gpts.listplaces.model.Place
+import com.discordtime.gpts.tools.Constants.PLACES_COLLECTION
+import com.google.firebase.firestore.CollectionReference
+import com.google.firebase.firestore.FirebaseFirestore
 
-class FirestoreAPI<T>: RemoteAPI<T> {
+class FirestoreAPI: RemoteAPI<Place> {
```

Comment:
I think we should not change this from generics for Place. I think we should leave it as T, return the snapshot and have a mapper that would transform the snapshot on the entity we want.